rust/perspective-viewer/test/js/migrate_viewer.spec.ts

```diff
@@ -342,44 +494,39 @@ test.describe("Migrate Viewer", () => {
     test.describe("Viewer config migrations", () => {
         for (const [name, old, current] of TESTS) {
             test(`Migrate '${name}'`, async ({ page }) => {
-                const converted = convert(
-                    JSON.parse(JSON.stringify(old), { warn: true }),
-                    {
-                        replace_defaults: true,
-                    }
-                );
+                const converted = convert(JSON.parse(JSON.stringify(old)), {
+                    replace_defaults: true,
+                    verbose: true,
+                });
                 expect(converted).toEqual(current);
             });
         }
     });
 
     test.describe("migrate", async () => {
```

Comment:
Previously this tested that 1) the migration script did what was expected and that 2) the viewer restored to the expected value. This caused a lot of noise when debugging, so the test has been reconfigured to only test 2) since we test 1) above.